docs/api/shell.md

```diff
@@ -20,13 +20,13 @@ The `shell` module has the following methods:
 
 * `fullPath` String
 
-Show the given file in a file manager. If possible, select the file.
+Show the given file in a file manager. If possible, select the file. Expects `fullPath` to be a fully-specified path (`/users/<username>/path/to/file` or `~/path/to/file`). Relative paths (`./file`) do not work.
```

Comment:
@kevinsawicki I'm afraid don't have access to a Windows machine. I can try it on Linux when I am home.

@baconface I'm aware that node can retrieve one's homedir. I ran into this behavior in a case where the first part of the file path (a "magic folder" somewhere on the user's system) is provided by a separate application. Not knowing the expected behavior (as well as observing that a `~`-based path succeeded with one call but not with another, with zero feedback) was a rather large pain in getting the two pieces to work together.